thop/vision/basic_hooks.py

```diff
@@ -12,11 +12,11 @@ def count_parameters(m, x, y):
     total_params = 0
     for p in m.parameters():
         total_params += torch.DoubleTensor([p.numel()])
-    m.total_params[0] = total_params
+    m.total_params[0] = counter_parameters(m.parameters())
```

Comment:
Why call count_parameters() here? `total_params` already provides the number of parameters of the model.